File: domusviewer/app/src/main/java/it/achdjian/paolo/domusviewer/DomusEngineRest/JSonDevice.java
package it.achdjian.paolo.domusviewer.DomusEngineRest;

import com.fasterxml.jackson.annotation.JsonIgnore;

import java.util.HashMap;
import java.util.Map;

import it.achdjian.paolo.domusviewer.zigbee.ZEndpoint;

/**
 * Created by Paolo Achdjian on 18/04/16.
 */
public class JSonDevice {
    public int short_address;
    public String extended_address;
    public int capability;
    public Map<Integer,String> endpoints = new HashMap<>();
}

File: temperature_monitor/app/src/main/java/it/achdjian/paolo/temperaturemonitor/domusEngine/rest/JsonDevice.kt
package it.achdjian.paolo.temperaturemonitor.domusEngine.rest

/**
 * Created by Paolo Achdjian on 7/9/17.
 */
data class JsonDevice(var short_address: Int, var extended_address: String, var capability: Int, var endpoints: Map<Integer,String>  ) {
}